portfolio/src/main/webapp/script.js

```diff
@@ -21,8 +21,36 @@ function addRandomGreeting() {
   // Picks a random quote and adds it to the page
 }
 
-function getData() {
-  fetch('/data').then(response => response.text()).then((msg) => {
-    document.getElementById('quote-container').innerText = msg;
+function getValues() {
```

Comment:
It might also be good to add some checking or fallback in case there aren't 3 messages in the response. If I understand correctly, currently, it would just raise an index out of bounds exception.

Reply:
Correct, I will consider implementing more logic to account for all cases.